tests/jobsautoreport/test_slack.py

```diff
@@ -70,7 +74,11 @@ def test_send_report_should_successfully_call_slack_api_with_expected_message_fo
             "type": "section",
             "text": {
                 "type": "mrkdwn",
-                "text": f"•\t _{report.number_of_e2e_or_subsystem_presubmit_jobs}_ jobs - :slack-green: {report.number_of_successful_e2e_or_subsystem_presubmit_jobs} :x: {report.number_of_failing_e2e_or_subsystem_presubmit_jobs}",
+                "text": (
+                    f"•\t _{report.number_of_e2e_or_subsystem_presubmit_jobs}_ jobs\n"
```

Comment:
```suggestion
                    f"•\t _{report.number_of_e2e_or_subsystem_presubmit_jobs}_ jobs in total\n"
```

just a suggestion, what do you think?

Reply:
:+1: